backend/plugins/erxes-agent_api/deploy/langfuse/.env.example

```
@@ -0,0 +1,54 @@
+# Central Langfuse v3 server config — copy to .env and CHANGE every secret.
+# Generate:  openssl rand -hex 32   (ENCRYPTION_KEY — must be 64 hex chars)
+#            openssl rand -base64 32 (SALT, NEXTAUTH_SECRET)
+
+# ── App (always required) ──────────────────────────────────────────────────
+LANGFUSE_PUBLIC_URL=https://langfuse.your-domain.com   # public URL of the UI
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)